.gitlab/release.yml

```
@@ -25,7 +25,7 @@ release:
         IMG_DESTINATIONS: agent-buildimages-rpm_x64:v$CI_PIPELINE_ID-${CI_COMMIT_SHORT_SHA},agent-buildimages-rpm_x64:latest
       - IMG_SOURCES: registry.ddbuild.io/ci/datadog-agent-buildimages/rpm_arm64:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}
         IMG_DESTINATIONS: agent-buildimages-rpm_arm64:v$CI_PIPELINE_ID-${CI_COMMIT_SHORT_SHA},agent-buildimages-rpm_arm64:latest
-      - IMG_SOURCES: registry.ddbuild.io/ci/datadog-agent-buildimages/linux-x64:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA},registry.ddbuild.io/ci/datadog-agent-buildimages/linux-arm64:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}
+      - IMG_SOURCES: registry.ddbuild.io/ci/datadog-agent-buildimages/linux-x64:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-x64,registry.ddbuild.io/ci/datadog-agent-buildimages/linux-arm64:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-arm64
```

Good catch, but I think this should even be
```suggestion
      - IMG_SOURCES: registry.ddbuild.io/ci/datadog-agent-buildimages/linux:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-x64,registry.ddbuild.io/ci/datadog-agent-buildimages/linux:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-arm64
```
(No arch in the image name, only in the tag)